distribution/docs/src/main/resources/content/_using/workspaces.adoc

```diff
@@ -0,0 +1,67 @@
+:title: Using Workspaces
+:type: using
+:status: published
+:parent: Using ${catalog-ui}
+:summary: Using workspaces within ${catalog-ui}
+:order: 03
+
+== {title}
+
+Within ${catalog-ui}, workspaces are collections of settings, searches, and bookmarks that can be shared between users and stored for repeated access.
```

Comment:
❓ What is a bookmark in the context of a workspace?

Reply:
I'm not sure at the moment